rviz_rendering_tests/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ if(BUILD_TESTING)
     set(DISPLAYPRESENT FALSE)
   endif()
 
-  if(APPLE OR DISPLAYPRESENT OR EnableDisplayTests STREQUAL "True")
+  if((APPLE AND NOT CMAKE_BUILD_TYPE STREQUAL "Debug") OR DISPLAYPRESENT OR EnableDisplayTests STREQUAL "True")
```

Comment:
That is now a very long line.

I think that's better to add a temporary extra conditional with a TODO as we expect it to disappear as soon as the tests are fixed in Debug. I assume that on the other hand the original `APPLE OR DISPLAYPRESENT OR EnableDisplayTests STREQUAL "True"` will stay as is for the foreseeable future, is that correct?

Reply:
I don't have strong feelings either way. The extra conditional optimized for preserving existing logic, this optimizes for minimal diff churn.